src/lib.rs

```diff
@@ -8,37 +8,49 @@ use goban::Goban;
 use crate::algorithm::Algorithm;
 use crate::bitboard::BitBoard;
 
-mod goban;
+mod algorithm;
 mod bitboard;
-mod stone;
+mod goban;
 mod node;
-mod algorithm;
 
 // Comment rajouter une fonction python sur rust
 // Simplement rajouter dans le block pymodule une fonction rust avec obligatoirement une instance Python<'_>, et si applicable un PyResult pour le retour
 // Presque n'importe quel type peut etre passé tant que c'est un type natif python/rust (check doc)
 // Pour compiler, maturin develop dans le terminal, qui genere un dylib dans le dossier target/debug qu'il faut mettre dans le dossier root du projet.
 
-const DEPTH: u32 = 1;
+const DEPTH: u32 = 9;
```

Comment:
Maybe `9` is too strong for us right now ^^' (just kidding, we can keep it that way).